README.md

```diff
@@ -179,7 +181,7 @@ func main() {
     //            Options include MaxRetry, Queue, Timeout, Deadline, Unique etc.
     // ----------------------------------------------------------------------------
 
-    c.SetDefaultOptions(tasks.ImageProcessing, asynq.MaxRetry(10), asynq.Timeout(3*time.Minute))
+    c.SetDefaultOptions(tasks.ImageProcessor, asynq.MaxRetry(10), asynq.Timeout(3*time.Minute))
```

Comment:
the first argument should be a task type, so it should be `tasks.TypeImageResize`

Reply:
I fixed it!